server/src/main/java/io/unitycatalog/server/persist/dao/TableInfoDAO.java

```diff
@@ -86,14 +80,14 @@ public static TableInfoDAO from(TableInfo tableInfo) {
   public TableInfo toTableInfo(boolean fetchColumns) {
     TableInfo tableInfo =
         new TableInfo()
-            .name(name)
+            .tableId(super.getId().toString())
```

Comment:
why do we call super, instead of getId/getName from the current instance? Subclasses might want to override the implementation. If not, we should make these methods `final`